### PR TITLE
Fix status server panic

### LINKED ISSUE #348: Kmesh panic when `curl 127.0.0.1:15200/bpf/kmesh/maps`
Kmesh panicked when execute debug by request the status server. 

**How to reproduce:**

run kmesh in workload mode, and then `curl 127.0.0.1:15200/bpf/kmesh/maps` in kmesh daemon.

More worse is that when after it panicked, it showed its still healthy

```
root@kurator-linux-0001:~# k get pod -n kmesh-system
NAME           READY   STATUS    RESTARTS   AGE
kmesh2-8nrwk   1/1     Running   0          11d

```

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #348

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/status/status_server.go [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+)
+
+var log = logger.NewLoggerField("status")
+
+const (
+	adminAddr = "localhost:15200"
+
+	patternHelp       = "/help"
+	patternOptions    = "/options"
+	patternBpfAdsMaps = "/debug/bpf/ads"
```
Comment (hzxuzhonghu) [inline pkg/status/status_server.go:41]: @nlgwcy Should we separate ads and workload here?

File: pkg/status/status_server.go [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+		return
+	}
+
+	w.WriteHeader(http.StatusOK)
+	cache := client.AdsController.Processor.Cache
+	dynamicRes := &adminv2.ConfigResources{}
+
+	dynamicRes.ClusterConfigs = cache.ClusterCache.DumpBpf()
+	dynamicRes.ListenerConfigs = cache.ListenerCache.DumpBpf()
+	dynamicRes.RouteConfigs = cache.RouteCache.StatusLookup()
```
Comment (nlgwcy) [inline pkg/status/status_server.go:108]: cache.RouteCache.StatusLookup() -> cache.RouteCache.DumpBpf

File: pkg/status/status_server.go (hunk 1 above)
Comment (nlgwcy) [reply pkg/status/status_server.go:41]: yes, need to be separated.

File: pkg/status/status_server.go (hunk 2 above)
Comment (hzxuzhonghu) [reply pkg/status/status_server.go:108]: missed it

**Pull request merged**